about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Piano Concert</title>
+</head>
+
+<body>
+    <header>
+        <nav class="first-nav">
+            <ul class="first-ul">
+                <li>
+                    <a href="#"><img src="./images/fb.png" alt="facebookIcon"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="./images/twtr.png" alt="twitter"></a>
+                </li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+
+            </ul>
+        </nav>
+        <nav class="lower-nav">
+            <span class="menu" id="menu">
+        <ul class="hamburger">
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+    </span>
+            <a href="index.html"><img class="logo-second" id="second" src="images/piano logo.png" alt="logo"></a>
+            <ul class="mobile-list" id="mob-menu">
+                <li><i class="fas fa-times close-button" id="close-button"></i></li>
+                <li><a href="index.html" class="navigation-list" id="about-link">Home</a></li>
+                <li><a href="#" class="navigation-list">Program</a></li>
+                <li><a href="#" class="navigation-list">Join</a></li>
+                <li><a href="#" class="navigation-list">Sponsor</a></li>
+                <li><a href="#" class="button navigation-list">Summer Concert</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="main about-cover" id="background">
+        <div class="about-mid">
+            <h1 class="about-h1">Hello Piano world!</h1>
```

Comment:
- [ ] You have not stuck to the design on the about page on the following fronts 
 - [ ]  Your colors are different from the design.
 - [ ] You have an h1 & h2 on the page but the design only has on heading in this section.
 
 **Design**
 
![des1](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/60b0970f-90ad-4f1a-92ef-1fea8b5e0d64)

**Your page**
![des2](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/8fa34f01-e7a1-4de4-8f96-5187b09ec27c)